### PR TITLE
[FIX] update docs and configs about openvino ssd deployment

### PR DESCRIPTION
**Pull request by q3394101**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

update docs and configs about openvino ssd deployment

## Modification

configs/mmdet/_base_/base_openvino_static-300x300.py
configs/mmdet/detection/detection_openvino_static-300x300.py
docs/en/backends/openvino.md

### CODE REVIEW

**Comment by AllentDan** [comment]
Hi, @q3394101. OpenVINO in `mmdeploy` only supports dynamic form actually. `opt_shapes` in configs are used for optimizing models. Maybe we can keep the 300x300 configs but rename them to dynamic ones.

**Comment by q3394101** [comment]
> Hi, @q3394101. OpenVINO in `mmdeploy` only supports dynamic form actually. `opt_shapes` in configs are used for optimizing models. Maybe we can keep the 300x300 configs but rename them to dynamic ones.



> Hi, @q3394101. OpenVINO in `mmdeploy` only supports dynamic form actually. `opt_shapes` in configs are used for optimizing models. Maybe we can keep the 300x300 configs but rename them to dynamic ones.

https://mmdeploy.readthedocs.io/en/latest/backends/openvino.html 
官方的ssd脚本无法运行，因为detection_openvino_dynamic.py不存在，所以我测试了static 300x300。

**Comment by AllentDan** [comment]
> 了

Thanks for correcting the typo in the docs. It could be missed during updating configs.

Static is not supported BTW.

**Comment by q3394101** [comment]
> Thanks for correcting the typo in the docs. It could be missed during updating configs.
> 
> Static is not supported BTW.
所以我可以在这里提docs和config pr吗？需要怎样的测试流程才可以呢？

File: configs/mmdet/_base_/base_openvino_static-300x300.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+_base_ = ['./base_static.py', '../../_base_/backends/openvino.py']
```
Comment (AllentDan) [inline configs/mmdet/_base_/base_openvino_static-300x300.py:1]: use `'./base_static.py'` to replace `'./base_dynamic.py'`

**Comment by AllentDan** [comment]
Hi, @q339410. Just rename the config file and change the base config inside it to the dynamic one.

File: configs/mmdet/_base_/base_openvino_static-300x300.py (hunk 1 above)
Comment (q3394101) [reply configs/mmdet/_base_/base_openvino_static-300x300.py:1]: shape怎么设置呀？还是继续用300x300吗？

Comment (q3394101) [reply configs/mmdet/_base_/base_openvino_static-300x300.py]: > Hi, @q339410. Just rename the config file and change the base config inside it to the dynamic one.

经过测试现在的可以跑300x300 dynamic，但是是否将300修改成其他的数值？如800x1344?

Comment (AllentDan) [reply configs/mmdet/_base_/base_openvino_static-300x300.py]: keeping the shape 300x300 looks good to me

Comment (q3394101) [reply configs/mmdet/_base_/base_openvino_static-300x300.py]: ok 已经提交了更改

File: configs/mmdet/_base_/base_openvino_dynamic.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+_base_ = ['./base_dynamic.py', '../../_base_/backends/openvino.py']
+
+onnx_config = dict(input_shape=None)
+
+backend_config = dict(
+    model_inputs=[dict(opt_shapes=dict(input=[1, 3, 300, 300]))])
```
Comment (AllentDan) [inline configs/mmdet/_base_/base_openvino_dynamic.py:6]: The file name should contain the opt shape info. Please append `300x300` to the file name.

File: configs/mmdet/detection/detection_openvino_dynamic.py [hunk 3]
```diff
@@ -0,0 +1 @@
+_base_ = ['../_base_/base_openvino_dynamic.py']
```
Comment (AllentDan) [inline configs/mmdet/detection/detection_openvino_dynamic.py:1]: The file name should contain the opt shape info. Please append `300x300` to the file name.

Comment (q3394101) [reply configs/mmdet/detection/detection_openvino_dynamic.py:1]: OK 已经修改 我发现doc已经改掉了

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by AllentDan** [comment]
Hi, @q3394101. No need to keep `configs/mmdet/_base_/base_openvino_dynamic.py` and `configs/mmdet/detection/detection_openvino_dynamic.py` as they are exactly the same as `300x300` configs

**Review by grimoire — APPROVED** [review]
LGTM

**Pull request merged**